sections/shared/Layout/AppLayout/Header/UserMenu/UserMenu.tsx

```diff
@@ -28,8 +32,12 @@ const UserMenu: FC = () => {
 	const { connectWallet } = Connector.useContainer();
 	const [settingsModalOpened, setSettingsModalOpened] = useState<boolean>(false);
 	const [uniswapWidgetOpened, setUniswapWidgetOpened] = useState<boolean>(false);
+	const [toggleTheme, setToggleTheme] = useState<boolean>(true);
```

Comment:
Can we only use `theme` to determine the icon and switch the value between `dark` and `light`?